Spielregeln_kurz.md

```diff
@@ -0,0 +1,10 @@
+# Vereinfachte Spielregeln 
+
+1. Zwei Teams werfen mit einem Ball auf eine Flasche, die in der Mitte steht.
+2. Geworfen wird von oben.
+3. Wenn ein Team eine die Flasche trifft, darf es trinken.
```

Comment:
```suggestion
3. Wenn ein Team die Flasche trifft, darf es trinken.
```

Reply:
Vielleicht ist es hier besser vom Ziel zu sprechen? SO wird die Flasche in der Mitte vorher ja auch bezeichnet.

Reply:
In den langen Regeln schon, aber in den vereinfachten Regeln spricht man oben ja auch von Flasche. Deshalb würde ich es (der Einfachheit halber) so lassen und nur den Schreibfehler korrigieren

Reply:
ach das ist ein anderes file :see_no_evil: